PatternPal/PatternPal.Core/Recognizers/AdapterRecognizer.cs

```diff
@@ -1,63 +1,59 @@
 ﻿#region
-
```

Comment:
All other files reviewed by me have a whiteline after `#region` and before `#endregion`, and possibly more to pair `using`-directives if it is a significantly longer list. I suggest to keep them here as well.